src/kit.ts

```diff
@@ -628,7 +649,15 @@ export async function getVSKitEnvironment(kit: Kit): Promise<Map<string, string>
   console.assert(kit.visualStudio);
   console.assert(kit.visualStudioArchitecture);
   const installs = await vsInstallations();
-  const requested = installs.find(inst => inst.instanceId == kit.visualStudio);
+  const match = (inst: VSInstallation) =>
+      // old Kit format
+      (inst.instanceId == kit.visualStudio) ||
```

Comment:
Actually, this is wrong.  `kit.visualStudio` will be in `VisualStudio.${majorVersion}.0` format in the old schema